src/components/modal/main/settings/m-settings.tsx

```diff
@@ -0,0 +1,74 @@
+import { FC } from 'react';
+import { MSettingsProps } from './types';
+import { Label, LabelSize } from '../../../typography';
+import { Input, InputType } from '../../../form';
+import { IconEye } from '../../../../elements';
+import clsx from 'clsx';
+
+export const MSettings: FC<MSettingsProps> = ({
```

Comment:
Maybe rename this to ModalSettingsTemplate or something like this. And add a comment so Smartive knows that this is a component only used for storybook.